test/worker_tools/basics_test.rb

```diff
@@ -122,6 +122,16 @@ def run
       assert_includes err.message, err_message
       assert import.failed?
     end
+
+    it 'sets the model to running state' do
```

Comment:
Shouldn't the status be set to "running" only during processing? Afterward, it should be set to one of the other options again?!